react/features/base/participants/subscriber.ts

```diff
@@ -3,25 +3,66 @@ import _ from 'lodash';
 import { IStore } from '../../app/types';
 import { getCurrentConference } from '../conference/functions';
 import {
-    getMultipleVideoSendingSupportFeatureFlag
+    getMultipleVideoSendingSupportFeatureFlag,
+    getSsrcRewritingFeatureFlag
 } from '../config/functions.any';
+import { VIDEO_TYPE } from '../media/constants';
 import StateListenerRegistry from '../redux/StateListenerRegistry';
 
 import { createVirtualScreenshareParticipant, participantLeft } from './actions';
+import { getRemoteScreensharesBasedOnPresence } from './functions';
 import { FakeParticipant } from './types';
 
 StateListenerRegistry.register(
     /* selector */ state => state['features/base/tracks'],
     /* listener */(tracks, store) => _updateScreenshareParticipants(store)
 );
 
+StateListenerRegistry.register(
+    /* selector */ state => state['features/base/participants'],
```

Comment:
Changed this one to track changed to `state['features/base/participants'].remoteVideoSources`. No, these 2 selectors cannot be combined since the we still need the track based selector for detecting the local screenshare. The presence based selector is used only for remote screenshare when ssrc-rewriting is enabled.